src/Tools/NodeDocumentationMarkdownGenerator/Verbs/FromPackageOptions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using CommandLine;
+using CommandLine.Text;
+
+namespace NodeDocumentationMarkdownGenerator.Verbs
+{
+    [Verb("frompackage", HelpText = "Generate documentation from a package folder")]
+    internal class FromPackageOptions
+    {
+        [Option('i', "input", HelpText = "Package folder path")]
+        public string InputFolderPath { get; set; }
+
+        [Option('r', "references", HelpText = "Folder paths to dlls that are used as references in the nodes", Required = false)]
```

Comment:
this says folder paths, is it paths to folders of binaries or to binaries directly?